qiskit/optimization/algorithms/minimum_eigen_optimizer.py

```diff
@@ -131,6 +131,26 @@ def get_compatibility_msg(self, problem: QuadraticProgram) -> str:
         """
         return QuadraticProgramToQubo.get_compatibility_msg(problem)
 
+    @property
+    def min_eigen_solver(self) -> Optional[MinimumEigensolver]:
```

Comment:
```suggestion
    def min_eigen_solver(self) ->MinimumEigensolver:
```
Since this is not allowed to be an Optional on the constructor (nor in the setter that is added). I think this should always return a valid value, and never None, hence it should not be Optional